libs/googletest/Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY: all clean
+
+all: lib/libgtest.a
+
+lib/libgtest.a: lib/gtest-all.o
+	ar -rv lib/libgtest.a lib/gtest-all.o
+
+lib/gtest-all.o: lib/src/gtest-all.cc
+	g++ -I lib/include -I lib -c $< -o $@
+
+lib/src/gtest-all.cc: release-1.8.0.tar.gz
+	rm -rf lib
+	tar xfm release-1.8.0.tar.gz
```

Comment:
I could have just put `https://github.com/google/googletest/archive/master.zip` there, but not sure that would be a good solution.